### PR TITLE
Minor API cleanup

### PR DESCRIPTION
**Pull request by JamesNK**
https://github.com/aspnet/SignalR/issues/2131

Waiting on approval for merge into 2.1 RTM.

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedHubMessage.cs [hunk 1]
```diff
@@ -3,9 +3,8 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.AspNetCore.SignalR.Protocol;
 
-namespace Microsoft.AspNetCore.SignalR
+namespace Microsoft.AspNetCore.SignalR.Protocol
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedHubMessage.cs]: Why is this in protocol?

File: src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedMessage.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.AspNetCore.SignalR
+namespace Microsoft.AspNetCore.SignalR.Protocol
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedMessage.cs]: Why is this in protocol?

Comment (JamesNK) [reply src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedMessage.cs]: All the other message classes are in `Microsoft.AspNetCore.SignalR.Protocol`. It feels weird that these "internalish only" infrastructure classes for messages are in the root SignalR namespace.

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedMessage.cs]: This isn't a message type. It's sits above all protocols and that's why it's in the root. There's a public WriteAsync on HubConnectionContext that takes this in.

**Pull request merged**